file.go

```diff
@@ -750,7 +754,9 @@ func (f *File) calculateEntryHash(IsADV bool) int {
 			hash = hash + batch.GetADVControl().EntryHash
 		}
 	}
-	return hash
+
+	// Ensure the entry hash cannot exceed 10 digits
+	return hash % int(math.Pow(10, 10))
```

Comment:
We do this inside each `Batch` with `numericField` so we could do that again instead of this conversion. Thoughts?

https://github.com/moov-io/ach/blob/master/batch.go#L770-L776

Reply:
`numericField` does give us the result we're looking for. 
https://github.com/moov-io/ach/blob/master/converters_test.go#L91

Reply:
I was kind of curious about performance. I'm fine with using the latter approach used in `Batch`.
```go
//pkg: github.com/moov-io/ach
//cpu: Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
//BenchmarkModulo
//BenchmarkModulo-16    	1000000000	         0.2669 ns/op
//PASS
func BenchmarkModulo(b *testing.B) {
	x := int(math.Pow10(10))
	for i := 0; i < b.N; i++ {
		_ = i % x
	}
}

//pkg: github.com/moov-io/ach
//cpu: Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
//BenchmarkStringCov
//BenchmarkStringCov-16    	10427030	       117.8 ns/op
//PASS
func BenchmarkStringCov(b *testing.B) {
	converters := &converters{}
	for i := 0; i < b.N; i++ {
		s := converters.numericField(i, 10)
		_, _ = strconv.Atoi(s)
	}
}

```

Reply:
If anything, in a separate PR, I can create a helper function that'll take in an `(v int, maxDigits uint)` and return an `int` that can be used across Files and Batches; and from there we can use the implement the optimal conversion.

Reply:
Totally agree. We should truncate `int` values and then later convert them to strings.